server/routes/v1/deployment.ts

```diff
@@ -161,3 +165,59 @@ export const resetDeploymentApprovals = [
     return res.json(deployment)
   },
 ]
+
+export const fetchRawModelFiles = [
+  ensureUserRole('user'),
+  bodyParser.json(),
+  async (req: Request, res: Response) => {
+    const { uuid, version, fileType } = req.params
+    const deployment: any = await findDeploymentByUuid(req.user!, uuid)
```

Comment:
Don't set `deployment` to any, use the type from `findDeploymentByUuid`.  It will give a warning about possibly being undefined, which is a valid check you need to do.

Reply:
Done